### DISCUSSION TITLE
Missing device names in dropdown menu.

### DISCUSSION THREAD

**Discussion by bogdancordos** [librenms/docker#465] [Q&A] [answered]
Hi,

I'm using 24.9.0 docker image and all works great, however, I have a rather odd issue with two of the views:
Under "Overview" -> "MAC Address"  and "ARP Table" the name of the devices in the dropdown "All Devices" menu are missing or rather, not displayed as I can select an empty position and it will bring up the device that was supposed to be there.
![librenms_arp_table](https://github.com/user-attachments/assets/8adc1f5e-ec14-42b1-8108-bcd78e7c8a50)

Work fine for "IPv4 Address", IPv6 Address" and "FDB Table"
Any suggestion into resolving this will be much appreciated.

**Comment by laf** [comment, marked as answer]
This probably needs reporting to the main librenms repo, this is purely for the docker images

**Reply by bogdancordos** [reply]
I am using docker, latest image. I can't replicate the issue on a non docker install.
